chunkflow/chunk/image/adjust_grey.py

```diff
@@ -129,55 +151,91 @@ def grey_augment(img, max_level_change=0.15, max_window_change=0.15, max_log2gam
     change_window = np.random.rand() < window_prob
     change_gamma = np.random.rand() < gamma_prob
 
-    level = (value_range[0]+value_range[1])/2;
-    half_window = (value_range[1] - value_range[0]) / 2;
-    log2gamma = 0;
-    
+    level = (value_range[0] + value_range[1]) / 2
+    half_window = (value_range[1] - value_range[0]) / 2
+    log2gamma = 0
+
     if change_level:
-        level += 2*(np.random.rand() - 0.5)*max_level_change
+        level += 2 * (np.random.rand() - 0.5) * max_level_change
     if change_window:
-        half_window += 2*(np.random.rand() - 0.5) * max_window_change/2
+        half_window += 2 * (np.random.rand() - 0.5) * max_window_change / 2
     if change_gamma:
-        log2gamma += 2*(np.random.rand() - 0.5)*max_log2gamma_change
+        log2gamma += 2 * (np.random.rand() - 0.5) * max_log2gamma_change
 
     if change_level or change_window or change_gamma:
         if change_gamma:
-            target_range = [0,1]
+            target_range = [0, 1]
             # gamma adjustment only makes sense in [0,1] range in the conventional sense
         else:
             target_range = value_range
 
         # level/window adjustement using rescale()
-        img = rescale(img, level+np.array([-1,1])*half_window, target_range)
+        img = rescale(img, level + np.array([-1, 1]) * half_window,
+                      target_range)
         img = np.clip(img, *target_range, img)
 
         if change_gamma:  # perhaps gamma should always happen before window/level on the raw full range of values?
             img = adjust_gamma(img, 2**log2gamma)
-            
+
             # rescale back to original/desired pixel value representation
-            img = rescale(img, [0,1], value_range)
+            img = rescale(img, [0, 1], value_range)
 
     return img
 
 
+def normalize_section_shang(image, nominalmin, nominalmax, clipvalues):
+    assert nominalmin < nominalmax
+    assert image.ndim == 3
+    global_offset = image.global_offset
+    originaltype = image.dtype
+    arr = image.astype(np.float32)
+
+    # number of bits per voxel
+    nbits = np.dtype(originaltype).itemsize * 8
+    nominalmax = float(2**nbits - 1)
+
+    nominalmin = nominalmin if nominalmin is not None else 0.0
+    nominalmax = nominalmax if nominalmax is not None else nominalmax
```

Comment:
This overwrote the input argument while the original lines in #66 was pulling from the safely stored self.nominalmax. I'd rename it to default_nominalmax or something